src/helpers/figures/charts/chart_common.ts

```diff
@@ -388,3 +362,31 @@ export function getChartPositionAtCenterOfViewport(
 
   return position;
 }
+
+export function getChartAxisTitleRuntime(design?: AxisDesign):
+  | {
+      display: boolean;
+      text: string;
+      color?: string;
+      font: {
+        style: "italic" | "normal";
+        weight: "bold" | "normal";
+      };
+      align: "start" | "center" | "end";
+    }
+  | undefined {
+  if (design?.title?.content) {
+    const { content, color, align, italic, bold } = design.title;
+    return {
+      display: true,
+      text: content,
```

Comment:
To dynamically translate axis title extracted from the json files (in odoo)
```suggestion
      // axis title translatable terms are extracted from json files
      text: _t(content),
```